### DISCUSSION TITLE
Can you push 1.22 to DockerHub please?

### DISCUSSION THREAD

**Discussion by vtnate** [OpenModelica/OpenModelica#11535] [General]
Hello, thank you for all your work getting 1.22 released! This is fantastic and I'm eager to use it in production. Can you push the new version to DockerHub? Thank you!

**Comment by casella** [comment]
@sjoelund are you taking care of this?

**Comment by vtnate** [comment]
Thanks @casella. I check dockerhub at least daily to see if this has been pushed. Soon, I hope.

**Comment by vtnate** [comment]
Thank you!

**Comment by casella** [comment]
@vtnate all fine now?
